### PR TITLE
Rapid release patch to increase name field of ncbi_taxa_name table to 500 characters

### PR DESCRIPTION
**Pull request by ens-sb**
## Description

Due to the need to store a long author list, production increased the size of the name field in the ncbi_taxa_name table to `varchar(500)`. To accomodate this change we needed to increase the size of the corresponding field in the compara schema.

**Related JIRA tickets:**
- ENSCOMPARASW-6123

#### Change 1
Altered the type of the name field of the ncbi_taxa_name table to `varchar(500)`.

#### Change 2
Added the corresponding patch and the insert statement to register the patch under the name `patch_107_108_b.sql|ncbi_taxa_name_varchar500`.

#### Change 3

Patched the test databases.

## Testing

After patching the test databases the full set of unittest were run on both cp1 (mysql version 5.7.27) and cp3 (mysql version 5.6.36).

---

## PR review checklist

- [ ] Is the PR against an appropriate branch?
- [ ] Does the code adhere to coding guidelines?
- [ ] Does the code do what it claims to do?
- [ ] Is the code readable? Is it appropriately documented?
- [ ] Is the logic in the correct place?
- [ ] Was the code tested appropriately? Are there unit tests? Are unit tests self-contained and non-redundant?
- [ ] Did Travis CI pass for the code in the PR? Is Codecov acceptable based on the included/updated unit tests?
- [ ] Will the new code fail gracefully?
- [ ] Does the code follow good practice for writing performant code (e.g. using a database transaction rather than repeated queries outside of a transaction)?
- [ ] Does it bring in an unnecessary dependency?
- [ ] If you are reviewing a new analysis, is it future-proof and pluggable?
- [ ] Does the PR meet agile guidelines?

### CODE REVIEW

File: src/test_data/databases/vertebrates/table.sql [hunk 1]
```diff
@@ -590,14 +590,14 @@ CREATE TABLE `sequence` (
   `sequence` longtext NOT NULL,
   PRIMARY KEY (`sequence_id`),
   KEY `md5sum` (`md5sum`)
-) ENGINE=MyISAM DEFAULT CHARSET=latin1 MAX_ROWS=10000000 AVG_ROW_LENGTH=19000;
+) ENGINE=MyISAM AUTO_INCREMENT=10 DEFAULT CHARSET=latin1 MAX_ROWS=10000000 AVG_ROW_LENGTH=19000 ROW_FORMAT=DYNAMIC;
```
Comment (twalsh-ebi) [inline src/test_data/databases/vertebrates/table.sql:593]:
```suggestion
) ENGINE=MyISAM DEFAULT CHARSET=latin1 MAX_ROWS=10000000 AVG_ROW_LENGTH=19000 ROW_FORMAT=DYNAMIC;
```

File: src/test_data/databases/vertebrates/table.sql [hunk 2]
```diff
@@ -160,7 +160,7 @@ CREATE TABLE `gene_member` (
   KEY `dnafrag_id_end` (`dnafrag_id`,`dnafrag_end`),
   KEY `biotype_dnafrag_id_start_end` (`biotype_group`,`dnafrag_id`,`dnafrag_start`,`dnafrag_end`),
   KEY `genome_db_id_biotype` (`genome_db_id`,`biotype_group`)
-) ENGINE=MyISAM DEFAULT CHARSET=latin1 MAX_ROWS=100000000;
+) ENGINE=MyISAM AUTO_INCREMENT=10 DEFAULT CHARSET=latin1 MAX_ROWS=100000000 ROW_FORMAT=DYNAMIC;
```
Comment (twalsh-ebi) [inline src/test_data/databases/vertebrates/table.sql:163]:
```suggestion
) ENGINE=MyISAM AUTO_INCREMENT=915675731 DEFAULT CHARSET=latin1 MAX_ROWS=100000000 ROW_FORMAT=DYNAMIC;
```

File: src/test_data/databases/vertebrates/table.sql [hunk 3]
```diff
@@ -566,22 +566,22 @@ CREATE TABLE `seq_member` (
   KEY `dnafrag_id_start` (`dnafrag_id`,`dnafrag_start`),
   KEY `dnafrag_id_end` (`dnafrag_id`,`dnafrag_end`),
   KEY `seq_member_gene_member_id_end` (`seq_member_id`,`gene_member_id`)
-) ENGINE=MyISAM DEFAULT CHARSET=latin1 MAX_ROWS=100000000;
+) ENGINE=MyISAM AUTO_INCREMENT=10 DEFAULT CHARSET=latin1 MAX_ROWS=100000000 ROW_FORMAT=DYNAMIC;
```
Comment (twalsh-ebi) [inline src/test_data/databases/vertebrates/table.sql:569]:
```suggestion
) ENGINE=MyISAM AUTO_INCREMENT=952569014 DEFAULT CHARSET=latin1 MAX_ROWS=100000000 ROW_FORMAT=DYNAMIC;
```

**Review by twalsh-ebi — COMMENTED** [review]
These changes look good to me. I just have some minor suggestions for changes to `AUTO_INCREMENT` values.

**Review by twalsh-ebi — APPROVED** [review]
Thanks @ens-sb for incorporating my feedback.

Travis checks are failing, but for reasons that are unrelated to this PR.

**Review by Simarpreet-Kaur-Bhurji — APPROVED** [review]

**Pull request merged**